CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-web-session ChangeLog
 
+### 1.6.1 - TBD
```

Comment:
This should be a breaking change reason being the errors  thrown by the two libraries are not identical. Other than that everything LGTM!

Reply:
do we care about session errors though? I'll look into this.

Reply:
if this library could throw an axios or http-client error that is part of the public API surface and someone may have written a test against anywhere downstream, this would be a breaking change.

If an http-client error is thrown internally and wrapped into some other error and is not part of the public API, then that's a different case.

Reply:
We should make sure to mention this in the changelog too.

Reply:
make it a major release here: https://github.com/digitalbazaar/bedrock-web-session/pull/12/commits/7add4e43549db8089cf69a26daba4220f1e78d9a